crates/consensus/src/hardforks/isthmus.rs

```diff
@@ -0,0 +1,78 @@
+//! Module containing a [Transaction] builder for the Isthmus network upgrade transactions.
+//!
+//! Isthmus network upgrade transactions are defined in the [OP Stack Specs][specs].
+//!
+//! [specs]: https://specs.optimism.io/protocol/isthmus/derivation.html#network-upgrade-automation-transactions
+//! [Transaction]: alloy_consensus::Transaction
+
+use crate::UpgradeDepositSource;
+use alloc::{string::String, vec::Vec};
+use alloy_eips::eip2718::Encodable2718;
+use alloy_primitives::{address, Address, Bytes, TxKind, B256, U256};
+
+use crate::{Hardfork, TxDeposit};
+
+/// The Isthmus network upgrade transactions.
+#[derive(Debug, Default, Clone, Copy)]
+pub struct Isthmus;
+
+impl Isthmus {
+    /// EIP-2935 From Address
+    pub const EIP2935_FROM: Address = address!("E9f0662359Bb2c8111840eFFD73B9AFA77CbDE10");
```

Comment:
Wrong address per the spec, should be `0x3462413Af4609098e1E27A490f554f260213D685`.

Reply:
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2935.md#deployment to save a few clicks